packages/core/upload/admin/src/utils/tests/containsAssetFilter.test.ts

```diff
@@ -14,7 +14,7 @@ describe('containsAssetFilter', () => {
         filters: {
           $and: [
             {
-              mime: 'image',
+              mime: { $eq: 'image' },
```

Comment:
This isn't just a type change, so why did this test pass before?

Reply:
Yes, I agree with you, I changed the Query type now and also the old unit test code works now